doc/user/content/integrations/websocket-api.md

```diff
@@ -69,6 +70,7 @@ To authenticate using a token, send an initial text or binary message containing
 ```
 {
 	"token": "<Your access token>",
+    "options": { <Optional map of run-time settings> }
```

Comment:
Both of these new options fields also need to go into the typescript definition, along with some more descriptive prose and maybe an example showing how to set one. (Again, totally fine to do docs in another PR.)

Reply:
Sounds good! I'll put these in a followup PR